### PR TITLE
Computer feature kelsie

### PR DESCRIPTION
**Pull request by KelsiePorter**
I did a rebase to grab your work from the game-feature branch and updated the game.rb file and a very minor attr_reader fix on the player.rb file. In MY version of the game.rb file, I created and tested the setup method and it works! I added a number of comments in the code for clarity. I did not make any changes to your branch but if you approve this pull request, we could both start working off of these updated files.

### CODE REVIEW

File: lib/cell.rb [hunk 1]
```diff
@@ -25,6 +25,7 @@ def fire_upon
     @fired_upon = true
     if @ship 
       @ship.hit
+      "Hit!"
```
Comment (JustJakeSeymour) [inline lib/cell.rb:28]: nice.

File: lib/computer.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    valid_coordinate = fired_upon_coordinate(chosen_coordinate_to_fire_upon)
+    
+    player_board.cells[valid_coordinate].fire_upon
+    # The above will either return "Hit!" or "Miss!" from the fire_upon method in the cell class
+  end
+
+  private
+  # every method that is written under this 'private' flag means that it can ONLY be accesses in this class.
+  # Methods written under the private flag are helper methods
```
Comment (JustJakeSeymour) [inline lib/computer.rb:51]: okay, thanks for telling me this, do you think we should put other helper methods under private in our other .rb files?

File: lib/game.rb [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+  end
+
+  def setup_text
+
+  end
+
+  private 
+
+  # Created helper method to use in #setup
```
Comment (JustJakeSeymour) [inline lib/game.rb:71]: nice!!

File: lib/player.rb [hunk 4]
```diff
@@ -1,6 +1,6 @@
 class Player
 
-  attr_reader :board, :ships
+  attr_reader :board, :cruiser, :submarine, :ships
```
Comment (JustJakeSeymour) [inline lib/player.rb:3]: oh, good catch here.

**Review by JustJakeSeymour — APPROVED** [review]

File: lib/computer.rb (hunk 2 above)
Comment (KelsiePorter) [reply lib/computer.rb:51]: Yes, I will update the help methods in the board class now

**Pull request merged**